Problem while deleting the dumped data from MongoDB

Although everything works fine on my local machine, while doing the Django Tests with GitHub Actions there is an error while destroying the database created for the tests. Right on the last line of the Djongo code when it is deleting the db on `creation.py` at line 30, Python can't delete the directory tree where the dumped data supposedly is with the path `dump`.

I am using Python version `3.9` and the MongoDB Docker image `mongo:4.4.4`.



#### Traceback
<img width="1259" alt="Screenshot 2021-04-14 at 17 25 32" src="https://user-images.githubusercontent.com/44342122/114745294-6d2efe80-9d46-11eb-9a4e-6714dad2352f.png">

```
Run make test
env/bin/python manage.py test --verbosity=0 --parallel --failfast
This version of djongo does not support "NULL, NOT NULL column validation check" fully. Visit https://nesdis.github.io/djongo/support/
Traceback (most recent call last):
This version of djongo does not support "schema validation using CONSTRAINT" fully. Visit https://nesdis.github.io/djongo/support/
  File "/home/runner/work/rest_api/rest_api/manage.py", line 22, in <module>
This version of djongo does not support "schema validation using KEY" fully. Visit https://nesdis.github.io/djongo/support/
    main()
This version of djongo does not support "schema validation using REFERENCES" fully. Visit https://nesdis.github.io/djongo/support/
  File "/home/runner/work/rest_api/rest_api/manage.py", line 18, in main
This version of djongo does not support "COLUMN DROP NOT NULL " fully. Visit https://nesdis.github.io/djongo/support/
This version of djongo does not support "DROP CASCADE" fully. Visit https://nesdis.github.io/djongo/support/
    execute_from_command_line(sys.argv)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/commands/test.py", line 23, in run_from_argv
    super().run_from_argv(argv)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/base.py", line 328, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/core/management/commands/test.py", line 53, in handle
    failures = test_runner.run_tests(test_labels)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/test/runner.py", line 684, in run_tests
    old_config = self.setup_databases(aliases=databases)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/test/runner.py", line 604, in setup_databases
    return _setup_databases(
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/test/utils.py", line 177, in setup_databases
    connection.creation.clone_test_db(
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/django/db/backends/base/creation.py", line 216, in clone_test_db
    self._clone_test_db(suffix, verbosity, keepdb)
  File "/home/runner/work/rest_api/rest_api/env/lib/python3.9/site-packages/djongo/creation.py", line 30, in _clone_test_db
    shutil.rmtree('dump')
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/shutil.py", line 709, in rmtree
    onerror(os.lstat, path, sys.exc_info())
  File "/opt/hostedtoolcache/Python/3.9.4/x64/lib/python3.9/shutil.py", line 707, in rmtree
    orig_st = os.lstat(path)
FileNotFoundError: [Errno 2] No such file or directory: 'dump'
make: *** [Makefile:35: test] Error 1
Error: Process completed with exit code 2.
```

--- a/djongo/creation.py
+++ b/djongo/creation.py
@@ -9,12 +9,23 @@ class DatabaseCreation(BaseDatabaseCreation):
     def _clone_test_db(self, suffix, verbosity, keepdb=False):
         source_database_name = self.connection.settings_dict['NAME']
         target_database_name = self.get_test_db_clone_settings(suffix)['NAME']
+        try:
+            host = self.connection.settings_dict['CLIENT']['host']
+        except KeyError:
+            host = None
         client = self.connection.client_connection
         if not keepdb:
             self._destroy_test_db(target_database_name, verbosity)
             args = [
                 'mongodump',
                 '--quiet',
+            ]
+            if host is not None:
+                args += [
+                    '--host',
+                    host
+                ]
+            args += [
                 '--db',
                 source_database_name
             ]
@@ -22,7 +33,14 @@ class DatabaseCreation(BaseDatabaseCreation):
             args = [
                 'mongorestore',
                 f'dump/{source_database_name}',
-                '--quiet',
+                '--quiet'
+            ]
+            if host is not None:
+                args += [
+                    '--host',
+                    host
+                ]
+            args += [
                 '--db',
                 target_database_name
             ]

